spec/supabase_swift_v1.yml

```diff
@@ -0,0 +1,2687 @@
+        code: |
+          ```js
+          let url = try await supabase.auth
+                  .getOAuthSignInURL(
+                      provider: .google,
+                      redirectTo: URL(string: "https://example.com/welcome")!
+                    )
+          ```
+      - id: sign-in-with-scopes
+        name: Sign in with scopes
+        isSpotlight: false
+        description: |
+          If you need additional data from an OAuth provider, you can include a space-separated list of scopes in your request to get back an OAuth provider token.
+          You may also need to specify the scopes in the provider's OAuth app settings, depending on the provider. The list of scopes will be documented by the third-party provider you are using and specifying scopes will enable you to use the OAuth provider token to call additional APIs supported by the third-party provider to get more information.
+        code: |
+          ```js
+          let url = try await supabase.auth
+                    .getOAuthSignInURL(
+                      provider: .github,
+                      scopes: "repo gist notifications"
+                    )
+          ```
+  - id: sign-out
+    title: 'signOut()'
+    $ref: '@supabase/gotrue-js.GoTrueClient.signOut'
+    notes: |
+      - In order to use the `signOut()` method, the user needs to be signed in first.
+    examples:
+      - id: sign-out
+        name: Sign out
+        isSpotlight: true
+        code: |
+          ```js
+          let url = try await supabase.auth.signOut()
+          ```
+  - id: verify-otp
+    title: 'verifyOtp()'
+    $ref: '@supabase/gotrue-js.GoTrueClient.verifyOtp'
+    notes: |
+      - The `verifyOtp` method takes in different verification types. If a phone number is used, the type can either be `sms` or `phone_change`. If an email address is used, the type can be one of the following: `signup`, `magiclink`, `recovery`, `invite` or `email_change`.
+      - The verification type used should be determined based on the corresponding auth method called before `verifyOtp` to sign up / sign-in a user.
+    examples:
+      - id: verify-sms-one-time-password(otp)
+        name: Verify Sms One-Time Password (OTP)
+        isSpotlight: true
+        code: |
+          ```js
+          try await supabase.auth.verifyOTP(phone: "+13334445555",
+                                  token: "123456",
+                                  type: .sms)
+          ```
+      - id: verify-signup-one-time-password(otp)
+        name: Verify Signup One-Time Password (OTP)
+        isSpotlight: false
+        code: |
+          ```js
+             try await supabase.auth.verifyOTP(email: "example@example-email.com",
+                                  token: "123456",
+                                  type: .signup)
+          ```
+  - id: get-session
```

Comment:
Confirming these aren't missing from this somehow, but are just going to come later? 
(supabase-js on the right here)

<img width="1197" alt="CleanShot 2023-04-03 at 09 27 10@2x" src="https://user-images.githubusercontent.com/105593/229502657-992827fc-58fd-4253-8fff-ba20bf73e81e.png">

Reply:
I might have missed it. Let me take a look and update.

Reply:
Hmm code examples should be there. I'm seeing it on dark mode @J0 how about for you?

I'm updating some code there though which isn't correct.